eth-balance/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+#[rustfmt::skip]
+pub mod abi;
+#[rustfmt::skip]
+pub mod pb;
+
+use num_bigint;
+use substreams::{store, Hex};
+use substreams::scalar::BigInt;
+use substreams_ethereum::{pb::eth as pbeth};
+use substreams::store::{StoreSetRaw, StoreSet, StoreNew};
+use pb::eth_balance::v1::{EthBalanceChanges, EthBalanceChange};
+
+#[substreams::handlers::map]
+fn map_block_to_balance_changes(
+    block: pbeth::v2::Block
+) ->Result<EthBalanceChanges, substreams::errors::Error> {
+    let mut eth_balance_changes = EthBalanceChanges { items: vec![] };
+
+    for transaction in &block.transaction_traces {
+        for call in &transaction.calls {
+            for balance_change in &call.balance_changes {
```

Comment:
why not get the balance changes directly from `block.balance_changes`?

Reply:
`block.balance_changes` only returns the balance change information related to the transaction that mined the block.